### ISSUE TITLE
Two CREDENTIALS_TOKENs having the same CountryCode & PartyId? 

### ISSUE THREAD

**Issue by ahzf** [ocpi/ocpi#284]
What is the expected behavior, when a 2nd token is registered having the same CountryCode and PartyId as some existing/registered credentials token? Just saying that the combination of both _should_ be unqiue is undefined behaviour in a security critical situation.

I also do not see how I could stop someone else sending me wrong CountryCodes and PartyIds and by this getting access to information that was not intended for her/him.

Saying all this I really do not understand the security model of OCPI. Any normal process would first create a login and password/token/whatever and then attach EMPIds and CPOIds as meta data. Why should allowing externals to annouce any identity they want be a useful feature? Allowing externals to update their URLs is nevertheless a great idea :)

Just compare this with SSL/TLS-certificates: Would it be a good idea when you sign a certificate for someone, but (s)he can later still choose any common name (s)he wants?

**Comment by RobertdeLeeuw-IHomer** [comment]
I fully agree we need to improve the security section of OCPI. We now leave to much open to implementation.
I think for OCPI 3.0 we need to look at more industry standard ways of doing things.

for OCPI 2.x, I think we expect a good OCPI implementation to limit what a specific party is allowed to send/access, based on the information exchanged during the credentials handshake. If a company says they are NL/ABC but send/request information for BE/DEF, it is best to not allow this.
